.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules
 bundle.js
 tailwind.css
 *.pem
+package-lock.json 
```

Comment:
question (blocking): what's the intention with ignoring this? there's no top-level package.json file so i'm surprised a package lock file shows up here.

Reply:
I think that's supposed to be `*/package-lock.json`. Zara and I kept accidentally committing the package.json file so we decided to add it.

Reply:
Ah, yeah that's a bit of an annoyance but we do need to have a lock file to ensure dependencies don't change out from under us

Reply:
got it. I can revert it then. in that case, when do we check in the lockfile? are we periodically updating it?

Reply:
When we explicitly update a dependency version or add/remove a dependency